VoxelSniperCore/src/main/java/com/github/kevindagame/brush/VoltMeterBrush.java

```diff
@@ -43,14 +43,28 @@ private void volt(final SnipeData v) {
         v.sendMessage(Messages.FACE_POWER_MESSAGE.replace("%face%", "South").replace("%direct%", "" + block.isBlockFacePowered(BlockFace.SOUTH)).replace("%indirect%", "" + block.isBlockFaceIndirectlyPowered(BlockFace.SOUTH)));
     }
 
+    @Override
+    protected boolean actPerform(SnipeData v) {
+        switch (this.snipeAction) {
+            case ARROW:
+                volt(v);
+                return true;
+            case GUNPOWDER:
+                data(v);
+                return true;
+            default:
+                return false;
+        }
+    }
+
     @Override
     protected final void arrow(final SnipeData v) {
-        this.volt(v);
+        positions.add(getTargetBlock().getLocation());
```

Comment:
Do we need to call the event for this? don't think we change any blocks?

Reply:
I have gone off the assumption that you may only inspect a block that is on your plot, so you want this validation